data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+			`	She pauses and looks at you seriously.`
+			`	"We think it would be prudent for you to represent your people as our point of contact. Would you accept such a role?"`
+			`	You give it a moment of thought.`
+			choice
+				`	"It certainly seems rather practical."`
+			`	"Fantastic," she says, though her tone is much more business like than the wording would imply, "In that case, if you could please take this to Earth and communicate with your various governments we can begin a dialogue on how best to deal with this current scenario."`
+			choice
+				`	"Various? We only have two governments."`
+			`	"Oh," she says, "we've treated the Syndicate as a de facto government for many decades now; with the amount of influence they have it was the only prudent thing to do."`
+			`	You take the boxed item with you and an escort awaits to return you to your ship. As the door behind you closes you are sure you hear Sayari mutter to someone, "It was really much easier when they just had one government though, what a mess."`
+				accept
+	
+	
+mission "Hai Leaks Response 2"
+	landing
+	name "Collect Representatives"
+	description "Collect human government representatives from <stopovers> and take them to <destination>."
+	source "Earth"
+	stopover "Deep"
+	stopover "Foundry"
+	destination "Hai-home"
+	blocked "You have a mission here for the Hai, but you need three bunks free."
+	clearance
+	to offer
+		has "Hai Leaks Response 1: done"
+	passengers 3
+	on offer
+		event "pirates respond to leaks"
+		event "hai ladybug available"
+		log "People" "Xilin Yang" `Deputy Minister Yang works in the Foreign Affairs department of the Republic government. He is a pleasant and astute man with a firm handshake. Handling the Hai secrecy crisis has become his job while the majority of the foreign ministry remains engaged with the fallout from the Free Worlds and the recent peace.`
+		conversation
+			`On approach to the spaceport on Earth you communicate your need to speak to an official of significant standing on diplomatic business. Since you are not a diplomat this does cause some minor kerfuffle in the process, but thanks to your renown from the Pug war you are able to blow past such petty objections sufficiently so as to shortcut the process. The diplomatic ID from the box Sayari gave you doesn't hurt either.`
+			`	The Republic Deputy Minister for Foreign Affairs meets you in a hired, private room at the spaceport.`
+			`	"Captain <last>," he steps around the table to grasp your hand in a firm handshake, "Xilin Yang, it's an honour to meet a war hero like yourself."`
+			choice
+				`	"It was a brutal business, but someone had to do it. For the people."`
```

Comment:
These choices are diametrically opposing attitudes, and they determine Xilin's predisposition towards the player. In this particular case the style is deliberate because I want the choice to slap the player with a "hey this choice as to how you represent yourseelf might mean something" warning.

Reply:
I see what you're saying, but is there a way to enforce that onto the player without a style that seems so... "over the top" for both choices, at least? I've updated the suggestion below to try to keep the direct opposedness but give players at least one way to move through the conversation without being so dramatic about their wording.

Reply:
These new versions work well enough. :)